scripts/index.js

```diff
@@ -111,24 +118,35 @@ $(document).keyup(function(event) {
 
                 shareRef[row - 1][i - 1] = "R";
             }
-            //this is for fun.
-            // document.getElementsByName(name)[0].style.backgroundColor = '#AA4A44';
-            // if (Math.random() >= 0.5) {
-            //     document.getElementsByName(name)[0].style.backgroundColor = '#44aa5c';
-            // }
-            // if (Math.random() >= 0.5) {
-            //     document.getElementsByName(name)[0].style.backgroundColor = '#fceea7';
-            // }
-            
-            // elem
+        }
 
+        for (let i = 1; i <= 6; i++) {
+            let name = `r${row}c${i}`
+            let usedName = `used${input.toUpperCase().charAt(i-1)}`;
+            let p = i - 1; // this is ugly get charAt to work later...
+
+            let bool_yellow = shareRef[row - 1][i - 1] == "Y";
+            let bool_list = !(letterList.toLowerCase().includes(input.toLowerCase().charAt(p), 0));

```

Comment:
These and `usedName` are not used anymore after you put the logic in line/place. Nice solution!